### ISSUE TITLE
[Accessibility] Number of remaining characters is not announced when using Character Count.

### ISSUE THREAD

**Issue by lewisdorigo** [scottish-government-design-system/design-system#117]
## Description of the issue
When focusing on an input that uses the [Character Count](https://designsystem.gov.scot/components/character-count) component, the number of remaining characters isn't announced to screen reader users: only the total number of characters for the field is announced.

The number of remaining characters is only announced when beginning to modify content to the field.

This could be resolved by adding an additional `aria-describedby` onto the field which references the “You have XXX characters remaining” element.

## Steps to reproduce the issue
While using a screen reader, move focus to an input that uses the Character Count component.

## Expected behaviour
The number or remaining characters is announced

**Comment by jsutcliffe** [comment]
This will be fixed in the next release.

**Comment by jsutcliffe** [comment]
Fixed in [commit ee03ba9](https://github.com/scottish-government-design-system/design-system/commit/ee03ba9eca99dae0b9adeba6b0ad40a6ba00070d) and [commit e41daf9](https://github.com/scottish-government-design-system/design-system/commit/e41daf997f06b7ed00203e63ed1c561cfd965537). Released in [npm package version 2.3.0](https://www.npmjs.com/package/@scottish-government/design-system/v/2.3.0).
